src/meta_data.rs

```diff
@@ -34,14 +35,15 @@ pub struct DatenLordNode {
     pub max_available_space_bytes: i64,
     /// Max volumes per node
     pub max_volumes_per_node: i32,
-    // /// Node IP
-    // pub ip_address: String, // TODO: add node IP address
+    /// Node IP
+    pub ip_address: IpAddr,
```

Comment:
If we introduce ip_address here, do we have the plan to replace the node_id with this ip_address?

Reply:
ip address is used for grpc communication.( We may have other network solution in the future.)
Node id is used as identification of worker/controller.
They are for different purpose.

Reply:
Node ID is for logs and other human readable information, node IP is for gRPC connection, since K8S DNS doesn't resolve pod IP from pod name by default